### PR TITLE
Refactor exp

### PR DESCRIPTION
**Pull request by bhigy**
Parameters `seed` and `language` are now part of the config.
Removed unnecessary experiments.
Made the different experiments more consistent (e.g. using default configs where possible).
Using fixed label encoder.

### CODE REVIEW

**Comment by bhigy** [comment]
Merging as I need this to run some experiments, but comment if needed.

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
         PLATALEA_DATA_ROOT: ${{ github.workspace }}/data/flickr1d
       run: |
         pip install sklearn python-Levenshtein coverage nltk
-        coverage run experiments/basic-stack/run.py --epochs=1 --meta=dataset.json --audio_features_fn=mfcc_features.pt
+        coverage run experiments/basic-default/run.py --epochs=1 --meta=dataset.json --audio_features_fn=mfcc_features.pt
```
Comment (egpbos) [inline .github/workflows/pythonapp.yml:51]: Fine by me, but why default instead of stack? Can we remove one or the other experiment?

Comment (egpbos) [reply .github/workflows/pythonapp.yml:51]: Ah I see you removed basic-stack below :)

File: platalea/dataset.py [hunk 2]
```diff
@@ -55,8 +56,12 @@ def __init__(self, root, feature_fname, split='train', language='en',
             self.text_key = 'raw_jp'
         else:
             raise ValueError('Language {} not supported.'.format(language))
+        self.root = root
+        self.split = split
+        self.language = language
         root_path = pathlib.Path(root)
-        # Loading metadata
+        with open(root_path / 'label_encoders.pkl', 'rb') as f:
```
Comment (egpbos) [inline platalea/dataset.py:63]: Where is this pickle file saved?

**Review by egpbos — COMMENTED** [review]
Looks very good to me. One thing I'm wondering: do you now have a collection of config files somewhere to replace all the previously separate experiments you have now merged into single experiment scripts?

File: platalea/dataset.py (hunk 2 above)
Comment (bhigy) [reply platalea/dataset.py:63]: you can find a version under /home/bjrhigy/corpora/flickr8k/label_encoders.pkl or in flickr1d repo.

**Comment by bhigy** [comment]
> Looks very good to me. One thing I'm wondering: do you now have a collection of config files somewhere to replace all the previously separate experiments you have now merged into single experiment scripts?

Not for now. If there are specific variants of the experiments I want to keep track of, this will be added.

**Pull request merged**